systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -849,6 +849,26 @@ public DoneableKafkaUser tlsUser(String clusterName, String name) {
                 .build());
     }
 
+    public DoneableKafkaUser quotasUser(String clusterName, String name, Integer prodRate, Integer consRate, Integer requestPerc) {
```

Comment:
Parameter `name` would be better called `userName`.
And the method is probably better called `userWithQuota`.